editor/src/components/canvas/context-menu-state.tsx

```diff
@@ -0,0 +1,41 @@
+import { atom, useAtom } from 'jotai'
+
+import { BodyMenuOpenClass } from '../../core/shared/utils'
+import React from 'react'
+
+export const ContextMenuStateAtom = atom([] as string[])
+export const ContextMenuOpen = atom((get) => {
+  const contextMenusOpen = get(ContextMenuStateAtom)
+  return contextMenusOpen.length > 0
+})
```

Comment:
Would love to use this state [here](https://github.com/concrete-utopia/utopia/blob/403e73d4da4a260fb41f19c229ace7f488e99867/editor/src/components/editor/editor-component-common.tsx#L57) (possible elsewhere) so I decided to put this in it's own derived atom. But I don't believe I am consume it there as the function isn't a hook and I didn't really wanna parameter drill it down. Open to feedback tho.

Reply:
We can use the atoms outside of React by using [stores](https://jotai.org/docs/guides/using-store-outside-react) -
```ts
const store = getDefaultStore()
const isMenuOpen = store.get(ContextMenuOpen)
```
and then we don't have to rely on class name for the keyboard event check

Reply:
Sick, this is exactly what I was looking for. Thanks @liady, def my first time using this library and atoms!